router/batchrouter/batchrouter.go

```diff
@@ -1324,7 +1324,6 @@ func (brt *HandleT) setJobStatus(batchJobs *BatchJobsT, isWarehouse bool, errOcc
 
 	// REPORTING - START
 	if brt.reporting != nil && brt.reportingEnabled {
-		types.AssertSameKeys(connectionDetailsMap, statusDetailsMap)
```

Comment:
keeping this would've helped validate these changes.

Reply:
done